### PR TITLE
Hw4

### PR DESCRIPTION
**Pull request by Green-Mad**
Завершення четвертого завдання.
Поки без зірочки.

### CODE REVIEW

File: Regex1.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+'''there is a string " name=Amanda=sssss&age=32&&salary=1500&currency=euro ". Convert this string to a dictionary {name:
+ Amanda, age: 32, salary: 1500, currency: euro}'''
+
+import re
+
+string = "name=Amanda=sssss&age=32&&salary=1500&currency=euro"
```
Comment (bohdandovbsyh) [inline Regex1.py]: it is not an original string from task

File: Regex3.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+text = "The Hubble.Space.Telescope (often referred to as HST or Hubble) is a space telescope that was launched into low Earth orbit in 1990 and remains in operation.... It was not the first space telescope, but it is one of the largest and most versatile, renowned both as a vital research tool and as a public relations boon for astronomy. The Hubble telescope is named after astronomer Edwin Hubble and is one of NASA's Great Observatories..... The Space Telescope Science Institute (STScI) selects Hubble's targets and processes the resulting data, while the Goddard Space Flight Center (GSFC) controls the spacecraft.A commission headed by Lew Allen, director of the Jet Propulsion Laboratory, was established to determine how the error could have arisen. The Allen Commission found that a reflective null corrector, a testing device used to achieve a properly shaped non-spherical mirror, had been incorrectly assembled—one lens was out of position by 1.3 mm (0.051 in)."
+text = re.sub(r'Hubble\.Space\.Telescope', 'Hubble Space Telescope', text)
+text = re.sub(r'\.{2,}', '.', text)
+sentences = re.split(r'(?<=[.!?])\s+', text)
+
+for sentence in sentences:
+    print(sentence)
```
Comment (bohdandovbsyh) [inline Regex3.py:22]: The Space Telescope Science Institute (STScI) selects Hubble's targets and processes the resulting data, while the Goddard Space Flight Center (GSFC) controls the spacecraft.A commission headed by Lew Allen, director of the Jet Propulsion Laboratory, was established to determine how the error could have arisen.


It is 2 sentences. But your code find it as 1

Comment (Green-Mad) [reply Regex3.py:22]: Fixed

File: Regex1.py (hunk 1 above)
Comment (Green-Mad) [reply Regex1.py]: Fixed

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**